### DISCUSSION TITLE
增加搜索页面的按时间过滤

### DISCUSSION THREAD

**Discussion by MerceMay** [the1812/Bilibili-Evolved#4945] [功能建议 / Ideas]
很多视频内容也具有时效性，希望能增加按时间过滤，或者按发布时间排序的功能

**Comment by the1812** [comment]
![Snipaste_2024-10-03_20-23-16](https://github.com/user-attachments/assets/8ed8b289-1621-4fd2-9add-a2731612617e)

**Comment by Geeyun-JY3** [comment]
破站网页端的搜索页最近好像加了按日期过滤的功能：
![image](https://github.com/user-attachments/assets/5a65a81c-4cc8-41ea-a541-d307002162f0)
